posts/2021/07/numpy-accessibility-guidelines.md

```diff
@@ -0,0 +1,180 @@
+<!--
+.. title: Making Numpy Accessible: Guidelines and Tools .. slug:
+numpy-accessibility-guidelines .. date: 2021-07-27 12:39:34 UTC-05:00 .. author:
+Mars Lee .. tags: Accessibility .. category: .. link: .. description: .. type:
+text
+-->
+
+![A large eye is placed over two bar graphs. Two silhouettes of heads are also
+overlaid the bar graphs.](/images/2021/07/numpy-accessibility/header_image.png)
+_Header illustration by author, Mars Lee_
+
+Numpy is now foundational to Python scientific computing. Our efforts reach
+millions of developers each month. As our user base grows, we recognize that we
+are neglecting the disabled community by not having our website and
+documentation up to modern accessibility standards.
+
+<!-- TEASER_END -->
+
+![The Venn diagram shows 2 intersecting circles, one labeled '10 million NumPy
+users' and the other labeled '15% of human population is disabled'. The area of
+intersection is labeled '1,500,000 disabled NumPy
+users.](/images/2021/07/numpy-accessibility/venn_diagram.png)
+
+As NumPy is early in the dependency chain, it has a huge install base. It is
+estimated that there are 10 million users of NumPy. Taking together another
+estimate that 15% of the human population disabled ([World Health
+Organization](https://www.who.int/disabilities/world_report/2011/report.pdf)),
+it is estimated that at least 1,500,000 NumPy users have a disability. Over a
+million users deserve an accessible NumPy website and documentation.
+
+In response to this, a small team started working on making NumPy’s more
+accessible, specifically its website and documentation. They weren’t experts in
+accessible technology. In fact, they feared that they didn’t know enough. Yet
+they strongly believed that accessible technology is a right every person should
+access and that NumPy could be accessible too.
+
+This small team started with Melissa Weber Mendonça and me, Mars Lee. Our
+previous skills and projects varied from working on documentation, front-end
+code, grant-writing, community work, software maintenance, and consulting work.
+
+We were very inspired by JupyterLab's big strides forward in accessibility this
+year. In the open-source ecosystem, accessibility is often considered
+low-priority, scary or someone else’s job. If JupyterLab could do it, so could
+NumPy. We decided to take this on.
+
+Making NumPy accessible is something that should have started a long time ago.
+However it  never quite started, mostly due to a lack of knowledge. Not everyone
+knows what accessible technology is, who it benefits, what guidelines to use or
```

Comment:
It may be worth noting that all these factors can be symptoms of or made worse by not having a community that includes disabled people. 

You can disregard this comment if you don't feel comfortable with it or think it fits you theme.

Reply:
Thank you for bringing that up! I thought about it but decided go in-depth with community aspect in the third blog post in this series. I wasn't sure how to bringing up community in the first blog post, since it requires more space to address it fully, so I decided keep the focus here on guidelines.